src/vs/base/parts/quickinput/browser/quickInput.ts

```diff
@@ -856,11 +871,12 @@ class QuickPick<T extends IQuickPickItem> extends QuickInput implements IQuickPi
 				hideInput = true;
 			}
 		}
-		this.ui.container.classList.toggle('hidden-input', hideInput);
+		this.ui.container.classList.toggle('hidden-input', hideInput && !this.description);
```

Comment:
Should that take the visibility of the check-all box into account? (Haven't check the layout.)

Reply:
Yeah you are right, will check that.

Reply:
Checked and it works